src/aio/usock_win.inc

```diff
@@ -535,15 +536,14 @@ DWORD nn_usock_open_pipe (struct nn_usock *self, const char *name)
     /*  First, create a fully qualified name for the named pipe. */
     _snprintf(fullname, sizeof (fullname), "\\\\.\\pipe\\%s", name);
 
-    /* TODO: Expose a way to pass lpSecurityAttributes */
     self->p = CreateFileA (
         fullname,
         GENERIC_READ | GENERIC_WRITE,
         0,
-        NULL,
+        &self->sec_attr,
         OPEN_ALWAYS,
         FILE_FLAG_OVERLAPPED,
-        NULL);
+        0);
 
     if (self->p == INVALID_HANDLE_VALUE)
         return GetLastError ();
```

Comment:
When `if (self->p == INVALID_HANDLE_VALUE)` is true, there's no pipe to clean up before return; seems fine as is.